python/libcuopt/CMakeLists.txt

```diff
@@ -46,15 +46,24 @@ FetchContent_Declare(
 )
 FetchContent_MakeAvailable(argparse)
 
+set(TBB OFF CACHE BOOL "Disable TBB")
```

Comment:
Since these are already added in cpp/Cmake, do we need to add here ?

Reply:
I think yes it was not propagated when I tested.